### PR TITLE
fix: change progress to percent in success props & warning in Progress

### LINKED ISSUE #25353: progress 在4.4版本警告
- [ ] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### Reproduction link
[Warning: [antd: Progress] `successPercent` is deprecated. Please use `success` instead. ](Warning: [antd: Progress] `successPercent` is deprecated. Please use `success` instead. )

### Steps to reproduce
Warning: [antd: Progress] `successPercent` is deprecated. Please use `success` instead. 

### What is expected?
消除警告，或者更新文档说明

### What is actually happening?
Warning: [antd: Progress] `successPercent` is deprecated. Please use `success` instead. 

| Environment | Info |
|---|---|
| antd | 4.4.0 |
| React | 16.13.1 |
| System | mac 10.15.4  |
| Browser | 83.0.4103.106 (正式版本) （64 位） |

### PR DESCRIPTION
**Pull request by fireairforce**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Perfermance optimization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link
close: https://github.com/ant-design/ant-design/issues/25353

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |    Fix Progress `success.progress` to `success.percent`.    |
| 🇨🇳 Chinese |   修正 Progress `success.progress` 为 `success.percent`。     |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Comment by fireairforce** [comment]
这个地方之前写错 devWarnning 的条件了😅

**Comment by fireairforce** [comment]
那个 api 如果也要调整的话，应该放在feature分支还是？

**Comment by afc163** [comment]
顺便把 `success.progress` 修正为 `success.percent` 吧。

1. `success.progress` 依旧有效，加个相应的警告。
2. 支持 `success.percent`。
3. 修正一下文档。

**Comment by afc163** [comment]
changelog 写一下。

File: components/progress/__tests__/__snapshots__/index.test.js.snap [hunk 1]
```diff
@@ -514,10 +506,6 @@ exports[`Progress render successColor progress 1`] = `
         class="ant-progress-bg"
         style="width: 60%; height: 8px;"
       />
-      <div
-        class="ant-progress-success-bg"
-        style="width: 30%; height: 8px; background-color: rgb(255, 255, 255);"
-      />
```
Comment (afc163) [inline components/progress/__tests__/__snapshots__/index.test.js.snap:520]: 为啥这个会变？

Comment (fireairforce) [reply components/progress/__tests__/__snapshots__/index.test.js.snap:520]: 还在修改，稍等~😂

**Comment by fireairforce** [comment]
Ok了，PTAL～

**Comment by fireairforce** [comment]
还得补下测试。。。QuQ

File: components/progress/__tests__/index.test.js [hunk 2]
```diff
@@ -166,4 +167,24 @@ describe('Progress', () => {
...
     );
   });
+
+  it('should warnning if use `progress` in success', () => {
+    const errorSpy = jest.spyOn(console, 'error').mockImplementation(() => {});
+    mount(
+      <Progress percent={60} success={{ progress: 30 }} />,
+    );
+    expect(errorSpy).toHaveBeenCalledWith(
+      'Warning: [antd: Progress] `progress` is deprecated. Please use `percent` instead.',
```
Comment (afc163) [inline components/progress/__tests__/index.test.js:177]:
```suggestion
      'Warning: [antd: Progress] `sucess.progress` is deprecated. Please use `sucess.percent` instead.',
```

Comment (fireairforce) [reply components/progress/__tests__/index.test.js:177]: done~

**Comment by zjscy666** [comment]
如何让这个警告消失，我用了customRequest

**Pull request merged**